Fix(chatlog): Match multi-character emoticons again

anthonybilinski opened the pull request:
Fixed after broken in https://github.com/qTox/qTox/pull/4940. Single-character UTF-8 emoji still work without whitespace on either side, but multi-character emoticon patterns like :smile: or :) do require surrounding whitespace, to avoid matching punctuation or HTML tags.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/qtox/qtox/4957)
<!-- Reviewable:end -->

sudden6 commented:
<img class="emoji" title=":lgtm_strong:" alt=":lgtm_strong:" align="absmiddle" src="https://reviewable.io/lgtm_strong.png" height="20" width="61"/>

---

Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/qtox/qtox/4957#-:-L5dlTFb37rLr9zmarCn:b-jbsf8z)*
<!-- Sent from Reviewable.io -->

anthonybilinski committed:
fix(chatlog): Match multi-character emoticons again

Fixed after broken in https://github.com/qTox/qTox/pull/4940. Single-character UTF-8 emoji still work without whitespace on either side, but multi-character emoticon patterns like :smile: or :) do require surrounding whitespace, to avoid matching punctuation or HTML tags.
